src/Sylius/Bundle/TaxonomyBundle/Doctrine/ORM/TaxonRepository.php

```diff
@@ -110,4 +110,24 @@ public function createListQueryBuilder()
     {
         return $this->createQueryBuilder('o')->leftJoin('o.translations', 'translation');
     }
+
+    /**
+     * @param QueryBuilder $qb
+     * @param null $locale
```

Comment:
Is the order important? IMHO null|string is more suitable as null is the default argument value.